src/test/java/core/basesyntax/service/impl/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvFileReaderImplTest {
+    CsvFileReaderImpl reader = new CsvFileReaderImpl();
```

Comment:
access modifer

Reply:
and let's initialize in @BeforeClass method

Reply:
+